bumble/transport/pyusb.py

```diff
@@ -205,16 +206,23 @@ async def close(self):
             await self.sink.stop()
             usb.util.release_interface(self.device, 0)
 
+    device_finder =  usb.core.find
```

Comment:
`usb_find` ?

Reply:
I wanted to share the "device_finder" between "usb.core.find" and "libusb_package.find".

Reply:
no I mean rename `device_finder` to `usb_find`